tests/unit/macaroons/test_security_policy.py

```diff
@@ -130,6 +131,53 @@ def test_identity_no_db_macaroon(self, monkeypatch):
         assert add_vary_cb.calls == [pretend.call("Authorization")]
         assert request.add_response_callback.calls == [pretend.call(vary_cb)]
 
+    def test_identity_frozen_user(self, monkeypatch):
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏we're evaluating `is_disabled` instead of `is_frozen` - which _should_ bubble up correctly in `is_disabled`. but I'd be remiss if I didn't point that out.

Reply:
https://github.com/pypi/warehouse/pull/14384/commits/1643d6d1f1d3f4a8d5c52a61f9d5456c27ddc6eb

We have coverage for frozen -> disabled elsewhere so I just updated the test name.